### PR TITLE
:penguin: sorted packages in dockerfiles and removed some duplicates

### LINKED ISSUE #786: Packages in dockerfiles should be sorted alphanumerically
**Is your feature request related to a problem? Please describe.**
The packages in the dockerfiles are not sorted alphanumerically, which makes comparisons, forks and pull requests harder than it should be

**Describe the solution you'd like**
Sort the packages alphanumerically e.g. with visual studio code (sort ascending ) ref https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

**Describe alternatives you've considered**
live with the pain ;)

**Additional context**

### PR DESCRIPTION
**Pull request by c0ffee**
Signed-off-by: Martin Schuessler <1407812+c0ffee@users.noreply.github.com>

**What this PR does / why we need it**:
Sorted packages in existing dockerfiles

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #786

### CODE REVIEW

File: images/Dockerfile.alpine-arm-rpi [hunk 1]
```diff
@@ -3,45 +3,37 @@ ARG BASE_IMAGE=alpine
...
-      tar \
-      rsync \
       bash-completion \
       blkid \
       busybox-openrc \
       ca-certificates \
+      connman \
       conntrack-tools \
       coreutils \
+      coreutils \
```
Comment (mauromorales) [inline images/Dockerfile.alpine-arm-rpi:14]: looks like we had duplications already

File: images/Dockerfile.alpine-arm-rpi [hunk 2]
```diff
@@ -3,45 +3,37 @@ ARG BASE_IMAGE=alpine
...
       cryptsetup \
       curl \
+      curl \
```
Comment (mauromorales) [inline images/Dockerfile.alpine-arm-rpi:17]: this one too

File: images/Dockerfile.alpine-arm-rpi [hunk 3]
```diff
@@ -3,45 +3,37 @@ ARG BASE_IMAGE=alpine
...
       dbus \
       dmidecode \
       dosfstools \
+      dosfstools \
+      e2fsprogs \
```
Comment (mauromorales) [inline images/Dockerfile.alpine-arm-rpi:22]: so do these ... will not overflow with more comments but I guess is the same in other files

**Review by mauromorales — CHANGES_REQUESTED** [review]
Thanks for this one, I agree it's so much better to consume when sorted. I'd say let's take the opportunity to remove any duplicated packages

**Comment by c0ffee** [comment]
good point i hope i got them now all

**Review by mauromorales — APPROVED** [review]
LGTM

**Review by Itxaka — APPROVED** [review]

**Comment by mudler** [comment]
Fails on https://github.com/kairos-io/kairos/issues/781 which is unrelated, merging! Thanks @c0ffee !

**Pull request merged**